### PR TITLE
Add board supermicro-x11ssh-f

### PR DESCRIPTION
**Pull request by root-hardenedvault**

### CODE REVIEW

File: config/coreboot-supermicro-x11ssh-f.config [hunk 1]
```diff
@@ -0,0 +1,888 @@
+#
```
Comment (tlaurion) [inline config/coreboot-supermicro-x11ssh-f.config:1]: Please make savedefconfig and replace this file by the content of defconfig outputted.

**Review by tlaurion — CHANGES_REQUESTED** [review]
Some changes required

**Comment by tlaurion** [comment]
Hello @root-hardenedvault ! Please see review!

**Comment by root-hardenedvault** [comment]
We won't upstream the kernel config since a few features from out-of-tree kernel has been adopted. Sorry, it's a wrong PR. Will make it public in our future repo though.

**Pull request closed without merging**